### PR TITLE
Changes:

### PR DESCRIPTION
**Pull request by Alma-hanif**
1. Added a module for pre-processing the text files using the ML library (nltk) in python
2. Updated Readme
3. Updated requirements.txt

### CODE REVIEW

**Comment by alexanderbarnhill** [comment]
Why put everything in the Scripts folder?

File: Scripts/pre_processing.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import os
```
Comment (Dimfred) [inline Scripts/pre_processing.py:1]: maybe rename `pre_processing` to `preprocessing`

Comment (Alma-hanif) [reply Scripts/pre_processing.py:1]: Thanks I will rename it . I tested it on a folder with 500 files at a time but yeah we can handle if the problem occurs later.

**Comment by Alma-hanif** [comment]
> Why put everything in the Scripts folder?
- I wanted the python files to be in a Script directory so that we can merge this branch to masters when the current topic of which data-source we will be using is resolved.
- But I have also moved back the test files for python to the 'tests' directory as you suggested

**Comment by alexanderbarnhill** [comment]
> > Why put everything in the Scripts folder?
> 
> * I wanted the python files to be in a Script directory so that we can merge this branch to masters when the current topic of which data-source we will be using is resolved.
> * But I have also moved back the test files for python to the 'tests' directory as you suggested

It would probably still be a good idea to keep the app.py in the main folder, because that is where the app is started.

**Pull request merged**